### ISSUE TITLE
Errors when interacting with mypy

### ISSUE THREAD

**Issue by DatGuy1** [pywinrt/python-winsdk#4]
There are a few issues when running mypy on stub files importing winsdk, but I'm not sure how feasible fixing them would be.

First type of error:

```python
from winsdk.windows.data.xml.dom import IXmlNode, XmlDocument
```
```
error: Module "winsdk.windows.data.xml.dom" has no attribute "IXmlNode"
error: Module "winsdk.windows.data.xml.dom" has no attribute "XmlDocument"
```

Second type, when adding ignore missing winsdk imports to a configuration file:

```
winsdk\windows\devices\sms\__init__.pyi:140: error: invalid syntax
```
Which seems to be because of https://github.com/pywinrt/python-winsdk/blob/main/pywinrt/winsdk/windows/devices/sms/__init__.pyi#L140 using the reserved 'from' keyword

**Comment by dlech** [comment]
> using the reserved 'from' keyword

Related issue: https://github.com/pywinrt/pywinrt/issues/6

**Comment by dlech** [comment]
This should be fixed in v1.0.0-beta.3.
